lib/cloud_formation/bridge/resources/elasti_cache_replication_group.rb

```diff
@@ -40,6 +40,8 @@ def create(request)
           }
         end
 
+        alias_method :update, :create
+
```

Comment:
Not for this one, re-creating a cache replication group isn´t really possible, you have kill the previous one and then create a new one.